drops/aio_quickstart_bash_scripts.json

```diff
@@ -0,0 +1,24 @@
+{
+  "Title": "Azure IoT Operations (AIO) Quickstart Bash Scripts",
+  "Summary": "A growing set of bash scripts to kickstart Azure IoT Operations in Github Codespaces.",
+  "Description": "This project provides a set of bash scripts to quickly set up and start Azure IoT Operations in Github Codespaces. These scripts are designed to automate the setup process, making it easier and faster to get started with Azure IoT Operations, or re-run scenarios and quickly move onto the next learning steps.",
+  "Authors": [
+    {
+      "Name": "Dan Benitah",
+      "Link": "https://www.linkedin.com/in/danbenitah"
+    }
+  ],
+  "Source": "https://github.com/danuw/aio-quickstart-bash-scripts",
+  "Type": "script_automation",
+  "Difficulty": "Medium",
+  "ProgrammingLanguage": [
+    "Bash"
+  ],
+  "Products": [
+    "Azure IoT Operations",
+    "Arc"
```

Comment:
Kubernetes? K3s? K8s? GitHub Codespaces?

Reply:
Do we have a list of the possible options to add? (will check other entries in the meantime)